test/test.js

```diff
@@ -0,0 +1,95 @@
+function test(name, fn) {
+	return require('tape')(name, { timeout: 1000 }, fn)
+}
+const mannish = require('../')
+
+test('Basic functionality: should call the right function and get the response', t => {
+	t.plan(2)
+
+	const mediator = mannish()
+
+	mediator.provide('book', function(book) {
+		t.equal(book, 'enchiridion')
+		return Promise.resolve('mathematical')
+	})
+
+	mediator.provide('unrelated book', () => t.fail(`This function shouldn't be called`))
+
+	mediator.call('book', 'enchiridion').then(response => {
+		t.equal(response, 'mathematical')
+		t.end()
+	})
+})
+
+test('Should receive a non-promise response, too', t => {
+	const mediator = mannish()
+
+	mediator.provide('book', () => 'mathematical')
+
+	mediator.call('book', 'enchiridion').then(response => {
+		t.equal(response, 'mathematical')
+		t.end()
+	})
+})
+
+test('Should receive rejected promise', t => {
+	const mediator = mannish()
+
+	mediator.provide('loot', function(location) {
+		t.equal(location, 'from the dungeon please')
+		return Promise.reject(new Error('skeleton'))
+	})
+
+	mediator.call('loot', 'from the dungeon please').catch(err => {
+		t.ok(err instanceof Error, 'err is an Error')
+		t.equal(err.message, 'skeleton')
+		t.end()
+	})
+})
+
+test(`Errors thrown in the provider should result in rejected responses`, t => {
```

Comment:
A reason to not use ticks everywhere is the confusion that goes along with it. If I'm messing with someone's code base, I try to check what type of quotes they're using, and use those. I would be surprised by people using ticks if they don't have variables `${var}`, and it would looks like a mistake, even though it works fine.

Does it really reduce cognitive overhead? I think it's negligible. It does have the upside that you can use apostrophes and double quotes without escaping them. It's pretty rare that I want a tick in a string compared to apostrophes and double quotes.

Anyway, :+1: for consistency. :grin: